tests/integration/stats.test.js

```diff
@@ -1,5 +1,4 @@
 const expect = require('expect');
-const get = require('lodash/get');
 const app = require('../../server');
 const {
   customerList,
```

Comment:
Sur le premier test, tu peux ajouter un saut de ligne entre le beforeEach et le it

Reply:
done